### ISSUE TITLE
Unable to install plugin on my Windows Surface 3

### ISSUE THREAD

**Issue by Jorj-X-McKie** [apprenticeharper/DeDRM_tools#31]
On installation I'm seeing 'file is invalid. It does not contain a top-level _init_.py file

here is a paste from details of error mssg

calibre, version 2.35.0
ERROR: Unhandled exception: <b>InvalidPlugin</b>:The plugin in u'C:/Users/Ivor/Downloads/DeDRM_tools_6.3.3.zip' is invalid. It does not contain a top-level __init__.py file

calibre 2.35 [64bit]  isfrozen: True is64bit: True
Windows-8-6.2.9200 Windows ('64bit', 'WindowsPE')
('Windows', '8', '6.2.9200')
Python 2.7.9
Windows: ('8', '6.2.9200', '', 'Multiprocessor Free')
Traceback (most recent call last):
  File "site-packages\calibre\gui2\preferences\plugins.py", line 316, in add_plugin
  File "site-packages\calibre\customize\ui.py", line 388, in add_plugin
  File "site-packages\calibre\customize\ui.py", line 53, in load_plugin
  File "site-packages\calibre\customize\zipplugin.py", line 188, in load
  File "site-packages\calibre\customize\zipplugin.py", line 286, in _locate_code
InvalidPlugin: The plugin in u'C:/Users/Ivor/Downloads/DeDRM_tools_6.3.3.zip' is invalid. It does not contain a top-level __init__.py file

**Comment by ElleKayEm** [comment]
That archive contains more than just the calibre plugin.  Unzip it and you will find a folder containing the DeDRM calibre plugin zip file.

**Comment by Jorj-X-McKie** [comment]
Thanks.... I eventually saw that there are nested zip files within the master file but still no luck so far even with correctly installed plugin. I'm reinstalling Calibre and trying current and older versions of plugins. I'm taking care to uninstall old plugin, restart app, then install plugin, test. Uninstall etc. BTW, I used to have this working perfectly on this same laptop running Win8 (but it wasn't easy as I recall). I recently installed Linux Mint and again struggling with it. I'll keep 'plugging away' at it until I get it working.
